std/js/Syntax.hx

```diff
@@ -54,4 +54,11 @@ extern class Syntax {
 	**/
 	@:overload(function(o:Dynamic, f:Int):Bool {})
 	static function delete(o:Dynamic, f:String):Bool;
+
+	/**
+		Generate `o[f]` expression
+	*/
+	static inline function field(o:Dynamic, f:String):Dynamic {
+		return code('{0}[{1}]', o, f);
```

Comment:
I would expect a method named `Syntax.field` generate `o.f`, not `o[f]`.

Reply:
I failed to find a better name.
Also it mimics well known `Reflect.field()` method.

Reply:
Perhaps you meant `o["f"]` , in case of something like `o["class"]` becoming `o.class`?

Reply:
`f` is an identifier here, so i meant `o[f]` without quotes.
Example:
```haxe
var fieldName:String = getFieldName();
Syntax.field(obj, fieldName); //generates obj[fieldName]
```